theano/compile/tests/test_function_module.py

```diff
@@ -266,7 +279,7 @@ def test_copy_share_memory(self):
             for key in storage_map_cpy.keys():
                 storage = storage_map_cpy[key]
                 if key not in i_o_variables or isinstance(key, theano.tensor.Constant):
-                    self.assertTrue(any([ storage is s for s in ori_storages]))
+                    self.assertTrue(any([storage is s for s in ori_storages]))
```

Comment:
`in` compares by value (like `==`), whereas the existing code compares by identity (with `is`).
I don't know the details of how `storage.__eq__` is implemented so that may be equivalent, but I would not do the change without being certain that it is.